app/controllers/support_interface/data_exports_controller.rb

```diff
@@ -30,7 +30,7 @@ def create
 
     def show
       @data_export = DataExport.find(params[:id])
-      @data_export_type = @data_export.name.parameterize.underscore
+      @data_export.update!(export_type: @data_export.export_type_value)
```

Comment:
I don't think this should be here. It's just setting a value that is used is the view right?

Reply:
Yeah this doesn't feel like the right place but it would fall over without it here. A Data Export now requires an `export_type` attribute to be populated each time it is generated

Reply:
Line 25 of the export should be 
```
  data_export = DataExport.create!(name: export_type.fetch(:name), initiator: current_support_user, export_type: export_type.fetch(:type) though i think
```